src/components/Playground/Playground.tsx

```diff
@@ -38,37 +98,77 @@ export default function Playground() {
+               
+                <div ref={cliCodeRef} className="h-80 bg-gray-900">
+                  <Cli decreaseCommandsLeft={decreaseCommandsLeft} />
+                </div>
+              </div>
+
+              <div className="flex justify-between">
+                <div className="w-[48%] border border-gray-300 dark:border-gray-700 rounded-lg p-4">
+                  <div className="flex items-center space-x-2 mb-2">
+                    <Clock className="text-red-600 dark:text-red-700" />
+                    <span className="font-semibold">Cleanup in:</span>
+                  </div>
+                  <div className="text-2xl font-bold">{formatTime(timeLeft)}</div>
+                  <div className="mt-2 bg-gray-200 dark:bg-gray-700 rounded-full h-2">
+                    <div
+                      className="bg-red-600 dark:bg-red-700 h-2 rounded-full"
+                      style={{ width: `${(timeLeft / (15 * 60)) * 100}%` }}
+                    ></div>
+                  </div>
+                </div>
+                <div className="w-[48%] border border-gray-300 dark:border-gray-700 rounded-lg p-4">
+                  <div className="flex items-center space-x-2 mb-2">
+                    <Command className="text-green-600 dark:text-green-600" />
+                    <span className="font-semibold">Commands left:</span>
+                  </div>
+                  <div className="text-2xl font-bold">{commandsLeft}</div>
+                  <div className="mt-2 bg-gray-200 dark:bg-gray-700 rounded-full h-2">
+                    <div
+                      className="bg-green-600 dark:bg-green-600 h-2 rounded-full"
+                      style={{ width: `${(commandsLeft / 1000) * 100}%` }}
+                    ></div>
+                  </div>
+                </div>
               </div>
             </div>
-            <div className="h-96 bg-gray-100 rounded-lg overflow-hidden shadow-md">
-              <Cli decreaseCommandsLeft={decreaseCommandsLeft}/>
-            </div>
-          </div>
-            <div className="flex flex-row justify-between text-gray-900 ">
-              <div className="mt-4 flex justify-between border border-gray-400 text-sm bg-transparent p-3 rounded-lg">
-                <span>Cleanup in : {formatTime(timeLeft)} mins</span>
+
+            <div className="space-y-4">
+              <div className="border  rounded-lg p-4">
+                <SearchBox search={search} setSearch={setSearch} />
               </div>
-              <div className="mt-4 flex justify-between border border-gray-400 text-sm bg-transparent p-3 rounded-lg">
-                <span>Command left: {commandsLeft}</span>
+              <div className="border rounded-lg p-4">
+                <h2 className="text-xl font-semibold mb-4">Recent Commands</h2>
+                <ul className="space-y-2 border">
+                  {['SELECT * FROM users', 'INSERT INTO products', 'UPDATE orders SET status = "shipped"'].map((cmd, index) => (
```

Comment:
Why's this required?

Reply:
We can use the browser session to populate recently run commands. I've just kept a placeholder array for now. Do you think this feature is worth implementing?